src/main/java/io/quarkus/gizmo2/creator/BlockCreator.java

```diff
@@ -2673,6 +2683,24 @@ default Expr mapOf(Expr... items) {
         return mapOf(List.of(items));
     }
     
+    /**
+     * Generate a call to {@link Optional#of()}.
+     * 
+     * @param value
```

Comment:
Missing description here.

Reply:
Hm, it's just the param of `Optional#of()`. I don't think it makes sense to add some superfluous description here, it's IMO obvious. BTW I did not add any desc for `mapOf()` entries either...